extra/helpers.py

```diff
@@ -1,6 +1,15 @@
 import multiprocessing, subprocess
 import cloudpickle  # type: ignore
 from typing import Any
+from tinygrad.nn.state import get_state_dict
+
+def get_net_size(model):
+  total_params, total_bytes = 0, 0
+  for param_name, param in get_state_dict(model).items():
+    if not ("num_batches_tracked" in param_name) and not ("running" in param_name):
```

Comment:
This still isn't generic though, I don't like the name gate. If it can't be generic maybe we don't want it.